indexing-service/src/main/java/org/apache/druid/indexing/seekablestream/SeekableStreamIndexTaskTuningConfig.java

```diff
@@ -293,6 +298,12 @@ public int getMaxColumnsToMerge()
     return maxColumnsToMerge;
   }
 
+  @JsonProperty
+  public boolean getReleaseLockOnHandoff()
```

Comment:
Nit: Primitive boolean getters are named `isXyz()` by convention.
Boxed `Boolean` getters may be named `getXyz()`.

```suggestion
  public boolean isReleaseLockOnHandoff()
```